package.json

```diff
@@ -26,7 +26,7 @@
     "split-e2e-tests": "yarn ts-node ./scripts/split-e2e-tests.ts && git add .circleci/config.yml",
     "pkg-clean": "rimraf build out pkg/node_modules pkg/yarn.lock",
     "pkg-install": "cd pkg && cp ../yarn.lock ./ && yarn --production",
-    "pkg-prune": "cd pkg && rimraf **/*.d.ts **/*.js.map **/*.d.ts.map **/README.md **/readme.md **/Readme.md **/CHANGELOG.md **/changelog.md **/Changelog.md **/HISTORY.md **/history.md **/History.md",
+    "pkg-prune": "cd pkg && rimraf **/*.d.ts **/*.js.map **/*.d.ts.map **/README.md **/readme.md **/Readme.md **/CHANGELOG.md **/changelog.md **/Changelog.md **/HISTORY.md **/history.md **/History.md && cp ../node_modules/typescript/lib/*.d.ts node_modules/typescript/lib/",
```

Comment:
why do the type def files need to be packaged? They shouldn't be used at runtime

Reply:
See my comment: https://github.com/aws-amplify/amplify-cli/pull/9207#issuecomment-985893700